torch/distributions/binomial.py

```diff
@@ -14,13 +14,13 @@ class Binomial(Distribution):
     Example::
 
         >>> m = Binomial(100, torch.tensor([0 , .2, .8, 1]))
-        >>> x = m.sample()
-        tensor([   0.,   22.,   71.,  100.])
+        >>> m.sample()
+        tensor([..., ..., ..., ...])
```

Comment:
this change is surprising

Reply:
That's because there is no way to do local random seeding. The output is random so we can't match it.

If I was doing this in numpy I'd do:

```
>>> import numpy as np
>>> rng = np.random.RandomState(0)
>>> rng.randint(0, 10, 5)
array([5, 0, 3, 3, 7])
```

and I wouldn't have to worry that the output would fail. Here I used the ellipsis matching inherited from the builtin doctest to match anything, but this either would need to be fixed by seeding the RNG or disabling output checking (which is easy with `# xdoctest: +IGNORE_WANT` - also not I'm open to adding an alias for that directive, I don't think `IGNORE_WANT` is the best name to indicate we aren't going to check the stdout / output.)

Reply:
Global random seeding (hidden from the docs) seems pretty reasonable to make the test work out. Although using the explicit RNG API would also work, we do have an obligation to show users the non-explicit rng state api as well in the docs.